addons/sale/models/sale_order_line.py

```diff
@@ -558,6 +558,19 @@ def _compute_pricelist_item_id(self):
                     date=line._get_order_date(),
                 )
 
+    def _reset_price_unit(self):
+        self.ensure_one()
+
+        line = self.with_company(self.company_id)
+        price = line._get_display_price()
+        product_taxes = line.product_id.taxes_id._filter_taxes_by_company(line.company_id)
+        line.price_unit = line.product_id._get_tax_included_unit_price_from_price(
+            price,
+            product_taxes=product_taxes,
+            fiscal_position=line.order_id.fiscal_position_id,
+        )
+        line.technical_price_unit = line.price_unit
```

Comment:
```suggestion
        price_unit = line.product_id._get_tax_included_unit_price_from_price(
            price,
            product_taxes=product_taxes,
            fiscal_position=line.order_id.fiscal_position_id,
        )
        line.update({
            'price_unit': price_unit,
            'technical_price_unit': price_unit,
        })
```
It'd be a bit more efficient to update both record fields at the same time.